Samples/BackgroundBlur/BackgroundBlur/TransformAsync_IMFAsyncCallback.cpp

```diff
@@ -0,0 +1,81 @@
+#include "TransformAsync.h"
+
+HRESULT TransformAsync::GetParameters(
+    DWORD* pdwFlags,
+    DWORD* pdwQueue)
+{
+    HRESULT hr = S_OK;
+
+    do
+    {
+        if ((pdwFlags == NULL) || (pdwQueue == NULL))
+        {
+            hr = E_POINTER;
+            break;
+        }
+
+        (*pdwFlags) = 0;
+        (*pdwQueue) = MFASYNC_CALLBACK_QUEUE_MULTITHREADED;
+    } while (false);
```

Comment:
Nit: If there is only one break point, maybe remove do-while statement to make it cleaner?